### DISCUSSION TITLE
Pathways for network shares

### DISCUSSION THREAD

**Discussion by Truk-22** [seanap/Plex-Audiobook-Guide#53] [Q&A] [answered]
Hi,

Thankyou for the great write up.

I am having a little issue setting up the pathways to folders for mp3 tag. I was able to set up the pull folder from my network drive correctly. But the save folders are not working. It is creating the folder structure under c:\"network location"\path to files.

This is what i am using

1=\\network name\\path to files\\%albumartist%\\%series%\\%year% - %album% [ '['%series% %series-part%']']\\%album% (%year%)[ '['%series% %series-part%']']$ifgreater(%_total_files%,1, - pt$num(%track%,2),)

Any help would be greatly apprectiated. 

Cheers
Truk.22

**Comment by seanap** [comment, marked as answer]
This is how mine looks, just replace `Z:\Audiobooks` with whatever the folder location you want to save them to.  Also make sure there are double escapes `\\`

`1=Z:\\audiobooks\\%albumartist%\\%series%\\%year% - %album% [ '['%series% %series-part%']']\\%album% (%year%)[ '['%series% %series-part%']']$ifgreater(%_total_files%,1, - pt$num(%track%,2),)`

I use map network drive to give my nas a letter and I use the absolute path not the network path

**Reply by Truk-22** [reply]
Thanks, 

Got it going via mapping the network drive.

All the best

Truk-22
